gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreStorageFactory.java

```diff
@@ -55,30 +55,31 @@
 
     private DatastoreWrapper datastore;
     private final boolean multitenant;
+    private final ColumnTypeRegistry<DatastoreColumnType> typeRegistry;
 
-    /**
-     * Creates new instance of {@code DatastoreStorageFactory}.
-     *
-     * @param datastore   the {@link Datastore} implementation to use
-     * @param multitenant if storage factory is configured to serve a multitenant application.
-     *                    This implementation does not support
-     *                    <a href="https://cloud.google.com/appengine/docs/java/multitenancy/multitenancy">Datastore
-     *                    recommended</a> way of creating multitenant apps yet.
-     *                    See {@link Storage#isMultitenant()}.
-     */
     @SuppressWarnings({"OverridableMethodCallDuringObjectConstruction", "OverriddenMethodCallDuringObjectConstruction"})
-    public DatastoreStorageFactory(Datastore datastore, boolean multitenant) {
+    private DatastoreStorageFactory(Builder builder) {
+        this.multitenant = builder.multitenant;
+        this.typeRegistry = builder.typeRegistry;
+        initDatastoreWrapper(builder.datastore);
+    }
+
+    @VisibleForTesting
+    @SuppressWarnings({"OverridableMethodCallDuringObjectConstruction", "OverriddenMethodCallDuringObjectConstruction"})
```

Comment:
Please try to fit 100 symbols wide.
Also, some comment for these suppressions is required.